bookings/views.py

```diff
@@ -1,3 +1,62 @@
-from django.shortcuts import render
+import json
+from operator   import attrgetter
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Min, Max
+
+from bookings.models  import Booking, BookingStatus
+from flights.models   import FlightDetail
+from core.utils       import check_access
+
+class MyBookingListView(View):
+    @check_access
+    def get(self, request):
+        try:
+            user       = request.user
+            status_id  = request.GET.get('status', 'upcoming')
+
+            status_set = {
+                'upcoming' : 1,
+                'last'     : 2,
+                'canceled' : 3,
+            }
```

Comment:
Enum 만들어 놓은거 활용하면 좋을 것 같아요
```suggestion
            status_set = {
                'upcoming' : BookingStatusEnum.UPCOMING.value,
                'last'     : BookingStatusEnum.LAST.value,
                'canceled' : BookingStatusEnum.CANCELED.value,
            }
```

Reply:
수정했습니다!